File: src/Lesson3/java/Ex2.java
package Lesson3.java;
import java.util.Scanner;

public class Ex2 {
    public static void main(String[] args){
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter number");
        int number = scanner.nextInt();

        boolean isPrime = true;
        for (int i = 2; i < number; i++){
            if (number % i == 0) {
                isPrime = false;
            }
        }
        if (isPrime){
            System.out.println("Prime number");
        } else {
            System.out.println("Not prime number");
        }
    }
}

File: src/Lesson3/forkotlin/Ex2.kt
package Lesson3.forkotlin

fun main() {

    print("Enter number: ")
    val number = readln().toInt()

    var isPrime = true

    if (number <= 1) {
        isPrime = false
    } else {

        for (i in 2 until number) {

            if (number % i == 0) {
                isPrime = false
                break
            }
        }
    }

    if (isPrime) {
        println("Prime number")
    } else {
        println("Not prime number")
    }
}